internal/config/db.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"log"
+
+	"github.com/b-open/jobbuzz/pkg/model"
+	"gorm.io/driver/mysql"
+	"gorm.io/gorm"
+)
+
+var Db gorm.DB
+
+// initialise DB once
+func InitDb() {
+	// TODO: 12-Factor the connection string
+	dsn := "jobbuzz:secret@tcp(127.0.0.1:3306)/jobbuzz?charset=utf8mb4&parseTime=True&loc=Local"
+	db, err := gorm.Open(mysql.Open(dsn), &gorm.Config{})
```

Comment:
I think this function's signature should return db and err.

So you can do `return gorm.Open(...)`